### ISSUE TITLE
skin tone analysis가 끝나지 않아요..

### ISSUE THREAD

**Issue by abcthing** [swsnu/swpp2019-team2#115]
급하게 노트북에 강아지 사진 있는 걸로 테스트해보았는데 5분동안 로딩만 이루어지고 있습니다.
사진이 문제였다면 에러메세지라도 띄우면 좋을 것 같아요!

**Comment by das425** [comment]
강아지로 하시면 얼굴 인식을 못해서 안되는 것 같습니다ㅠㅠ 사람 셀카로 하셔야 해요!

**Comment by abcthing** [comment]
네 지금 제 사진으로 해보았는데 빠르게 결과가 나옵니다! 인식을 못하는 경우 타임리밋(?)같은 제한 두셔서 에러 띄워주면 더 좋은 서비스가 될 것 같습니다 !

**Comment by das425** [comment]
네 감사합니다 :) 수정하도록 하겠습니다

**Comment by aeri206** [comment]
Tone Analysis 화면에 추가 설명 필요
 - (화면 밝기. 인물 사진 등)

Timeout 설정
- 너무 오래 돌 경우
